client/src/pages/Bookings/Bookings.tsx

```diff
@@ -0,0 +1,146 @@
+import { useState, useEffect } from 'react';
+import { Grid, Typography, Paper } from '@mui/material';
+import getRequests from '../../helpers/APICalls/getRequests';
+import updateRequest from '../../helpers/APICalls/updateRequest';
+import { Request } from '../../interface/RequestApiData';
+import Booking from './Booking/Booking';
+import Calendar from './Calendar/Calendar';
+import useStyles from './useStyles';
+import { RequestStatus } from '../../types/RequestStatus';
+
+export default function Sitters(): JSX.Element {
```

Comment:
```suggestion
export default function Bookings(): JSX.Element {
```
Wouldn't Booking/Request be a more suitable name for this component